<launch>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find arhex_control)/config/arhex_control.yaml" command="load"/>
  <!-- <param name="robot_description" textfile="$(find arhex_description)/urdf/arhex.urdf"/> -->
  <!-- load the controllers -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
    output="screen" ns="/arhex" args="torso_position_controller shoulder_1_position_controller shoulder_2_position_controller elbow_position_controller wrist_1_position_controller wrist_2_position_controller wrist_3_position_controller ef_1_position_controller ef_2_position_controller L_FR_position_controller L_MR_position_controller L_RR_position_controller L_RL_position_controller L_ML_position_controller L_FL_position_controller"/>

  <!--node pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" name="teleop" output="screen">
 </node-->

  <!-- convert joint states to TF transforms for rviz, etc -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
    respawn="false" output="screen">
    <remap from="/joint_states" to="/arhex/joint_states" />
  </node>
 <node name="gait_control_node" pkg="arhex_control" type="gait.py" respawn="true" />
 <!-- <node name="bound_encoder_node" pkg="arhex_control" type="bound_encoder.py" respawn="true" /> -->
</launch>
